code/modules/antagonists/revenant/revenant_abilities.dm

```diff
@@ -38,6 +38,7 @@
 		if(prob(10))
 			to_chat(target, span_revennotice("You feel as if you are being watched."))
 		return
+	src.log_message(span_warning("has started to harvest [target][(!target.mind) ? "(no mind)" : ""]."), LOG_ATTACK)
```

Comment:
```suggestion
	log_message(span_warning("has started to harvest [key_name(target)]."), LOG_ATTACK)
```
No need for src, and use the key_name() helper

Reply:
should be log_combat so both parties get the entry.